src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(
+    century
+      ? (person) => Math.ceil(person.died / 100) === century
+      && person.sex === 'm'
+      : (person) => person.sex === 'm'
+  );
```

Comment:
вынеси колбеки в переменные
```suggestion
  const callback1 = ... ;
  const callback2 = ... ;
  
  const men = people.filter(century ? callback1 : callback2);
```